### PR TITLE
Test branching

### PR DESCRIPTION
**Pull request by AnushHakobyan**

### CODE REVIEW

**Comment by Kamagh** [comment]
We can consider some slight modifications in the index.html file to make it much better.

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ Learning app aca course 2021
 Our first change after initial commit
 
 Change to have some update
+
```
Comment (AnushHakobyan) [inline README.md:7]: please remove this line

**Review by AnushHakobyan — COMMENTED** [review]
aaaa

File: README.md (hunk 1 above)
Comment (AnushHakobyan) [reply README.md:7]: I will not do this, OK?!!!

**Pull request merged**